docs/detailed-push-process.md

```diff
@@ -67,9 +67,9 @@ You can get the route of your application with `epinio apps list` or `epinio app
 
 During installation, if you specified a system domain using the `--system-domain` parameter, then your application routes will be sudomains of that domain.
 Epinio considers this domain to be a production server and thus creates a production level TLS certificate for your application using [Let's Encrypt](https://letsencrypt.org/).
-This happens using the [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) which is one more of the components Epinio installs with `epinio install`.
+This happens to use the [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) which is one more of the components Epinio installs with `epinio install`.
```

Comment:
This refers to the above sentence. Maybe `This works by using` is better?

Reply:
Works for me too

Reply:
Done: 7426621bb5468134276fb09ae8f76ed1e0379ac8